packages/discovery-provider/src/queries/download_csv.py

```diff
@@ -192,26 +192,28 @@ def download_purchases(args: DownloadPurchasesArgs):
     return to_download
 
 
-def format_sale_for_csv(result, seller_handle, seller_user_id, include_email=False):
+def format_sale_for_download(
```

Comment:
we can show this in a different way in the actual csv, this just makes stuff easier on the code side of things